src/archivematicaCommon/lib/mcpserver_cli.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python2
+
+from __future__ import print_function, unicode_literals
+
+import os
+
+import sys
+
+import mcpserver
+from protos import mcpserver_pb2
+
+
+def usage(commands):
+    script_name = os.path.basename(__file__)
+    sys.exit('{} [{}]'.format(script_name, '|'.join(commands)))
+
+
+if __name__ == '__main__':
+    commands = [
+        'ApproveJob',
+        'ApproveTransfer',
+        'ChoiceList',
+    ]
+
+    if len(sys.argv) == 1:
+        usage(commands)
+
+    cmd = sys.argv[1]
+    client = mcpserver.get_client()
+
+    if cmd not in commands:
+        usage(commands)
```

Comment:
You're right. I will do it. They were conceived for testing purposes but I think they can be handy. If `archivematica-common` becomes its own pypi package, these two could be defined as extra entry points. Eventually I think we should have just one public-facing CLI tool but we're still far from that.